apps/stable_diffusion/web/ui/img2img_ui.py

```diff
@@ -323,7 +326,7 @@ def img2img_api(
         InputData["seed"],
         batch_count=1,
         batch_size=1,
-        scheduler="EulerDiscrete",
+        scheduler="DEISMultistep",
```

Comment:
Can we leave this as EulerDiscrete for now ? We will need to update all the golden images otherwise. We can do that separately if we want.

Reply:
just img2img, or all the ones I changed to DEIS by default?

Reply:
all of them since I think we have golden tests for each. We can land a default scheduler change separately and update the goldens in one PR. We could also offload it to GPU if we want.